src/layers/layer-factory.js

```diff
@@ -240,6 +240,16 @@ export const LAYER_VIS_CONFIGS = {
     group: PROPERTY_GROUPS.height,
     property: 'sizeRange'
   },
+  coverageRange: {
+    type: 'number',
+    defaultValue: [0, 1],
+    label: 'Coverage Range',
```

Comment:
This could be clearer, perhaps `Coverage by value`, `Dynamic coverage`, etc. And static coverage could change to e.g. `Fixed coverage`, `Manual coverage`, etc.